src/test/java/teammates/test/cases/logic/TeamEvalResultTest.java

```diff
@@ -491,7 +491,7 @@ private void verifyNormalized(double[] expected, double[] input) {
                 Arrays.toString(TeamEvalResult.normalizeValues(input)));
     }
     
-    private static void showCalculationSteps(int[][] input) {
+    /*private static void showCalculationSteps(int[][] input) {
```

Comment:
Hmm, if we are just commenting this out, then maybe let's just suppress the rule here?